web/assets/styles/main.css

```diff
@@ -619,3 +631,9 @@ main nav {
   font-weight: 700;
   src: local('Roboto Condensed Bold'), local('RobotoCondensed-Bold');
 }
+@font-face {
+  font-family: 'Inconsolata';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Inconsolata Regular'), local('Inconsolata-Regular');
```

Comment:
If you have the fonts locally installed and are offline, things still look correct.  (This is particularly good for development.)